docker-compose.yml

```diff
@@ -125,20 +126,24 @@ services:
     volumes:
       - streamlit-cachedata:/root/.cache
   rubrix:
+    env_file: .env
+    container_name: ${USER_NAME-test}-rubrix
     image: recognai/rubrix:latest
     restart: unless-stopped
     ports:
       - "6900:80"
     environment:
       ELASTICSEARCH: http://elasticsearch:9200
       # RUBRIX_LOCAL_AUTH_USERS_DB_FILE: /config/.users.yaml
-      RUBRIX_LOCAL_AUTH_DEFAULT_PASSWORD: ${RUBRIX_LOCAL_AUTH_DEFAULT_PASSWORD-$$2y$$05$$X1i7R1R7CeFyD/dxZ5ttu.FcJQbL2Gh30.86OxCxzTXl/BR.RO.6O}
+      # RUBRIX_LOCAL_AUTH_DEFAULT_PASSWORD: ${RUBRIX_LOCAL_AUTH_DEFAULT_PASSWORD-$$2y$$05$$X1i7R1R7CeFyD/dxZ5ttu.FcJQbL2Gh30.86OxCxzTXl/BR.RO.6O}
     # volumes:
     #   - ./services/rubrix/src/config/.users.yaml:/config/.users.yaml
```

Comment:
can these be removed if unused?

Reply:
I set `team44` as the password, the default is `rubrix`. I'm happy with either.

Reply:
@diegoquintanav could you answer this one?

Reply:
that is an environment variable that, when passed to the rubrix container (documented by rubrix in https://rubrix.readthedocs.io/en/stable/getting_started/user-management.html?highlight=password#how-to-override-the-default-user-password)

if not set then I think the default password is `1234`. We can remove it or keep it, I'm fine with either.